packages/api/src/graphql/article.ts

```diff
@@ -268,7 +268,7 @@ export const GraphQLPublicArticle: GraphQLObjectType<
     socialMediaTitle: {type: GraphQLString},
     socialMediaDescription: {type: GraphQLString},
     socialMediaAuthors: {
-      type: GraphQLList(GraphQLAuthor),
+      type: GraphQLNonNull(GraphQLList(GraphQLAuthor)),
```

Comment:
Missing one `GraphQLNonNull` between list and author. same as with properties or tags.

Reply:
Done